src/ppl/src/IOPlacer.cpp

```diff
@@ -205,18 +204,20 @@ void IOPlacer::randomPlacement(std::vector<int> pin_indices, std::vector<int> sl
 
   std::vector<Slot> &slots = top_layer ? top_layer_slots_ : slots_;
 
+  std::vector<IOPin>& io_pins = netlist_.getIOPins();
```

Comment:
const?

Reply:
Not applicable here; I'm changing the items of this vector in the loop below.